package.json

```diff
@@ -36,7 +36,7 @@
         "portal-vue": "^2.1.7",
         "postcss-import": "^12.0.1",
         "resolve-url-loader": "^3.1.0",
-        "tailwindcss": "^1.9.2",
+        "tailwindcss": "^1.9.4",
```

Comment:
For future PRs please avoid bumping dependencies as part of a feature PR :) This will reduce the likelihood of having to deal with merge conflicts in the lock files

Reply:
Noted. Adding tailwind wasn't on purpose. I was tired of tinkering with the weird form submission bug so I just made the commit once I had something working.